app/src/main/res/values/strings.xml

```diff
@@ -175,4 +175,10 @@
     <string name="fork_me">Fork me</string>
     <string name="cd_arrowTagImage">Arrow Tagging Game</string>
     <string name="cd_qrCode">QR Code</string>
+    <string name="invitation_title">Invite your friends to GDG app</string>
+    <string name="invitation_message">Download GDG Android App and discover Google Developers Group content while being on the go.</string>
+    <string name="invitation_deep_link" translatable="false">https://developers.google.com/groups/</string>

```

Comment:
The PR was good like that. Just add it...


[image: --]
Friedger Müffke
[image: http://]about.me/friedger
<http://about.me/friedger>


2015-09-15 10:00 GMT+02:00 Said Tahsin Dane <notifications@github.com>:

> In app/src/main/res/values/strings.xml
> <https://github.com/gdg-x/frisbee/pull/489#discussion_r39483630>:
>
> > @@ -175,4 +175,10 @@
> >      <string name="fork_me">Fork me</string>
> >      <string name="cd_arrowTagImage">Arrow Tagging Game</string>
> >      <string name="cd_qrCode">QR Code</string>
> > +    <string name="invitation_title">Invite your friends to GDG app</string>
> > +    <string name="invitation_message">Download GDG Android App and discover Google Developers Group content while being on the go.</string>
> > +    <string name="invitation_deep_link" translatable="false">https://developers.google.com/groups/</string>
>
> Why did you merge it? :) I was gonna go over it one more time and addadd
> these event filters.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gdg-x/frisbee/pull/489/files#r39483630>.
>